src/acom_music_box/tools/waccmToMusicBox.py

```diff
@@ -112,14 +197,13 @@ def readWACCM(waccmMusicaDict, latitude, longitude,
     logger.info("whenStr = {}".format(whenStr))
     singlePoint = waccmDataSet.sel(lon=longitude, lat=latitude, lev=1000.0,
                                    time=whenStr, method="nearest")
-    if (True):
+    if (False):
         # diagnostic to look at single point structure
         logger.info("WACCM singlePoint = {}".format(singlePoint))
```

Comment:
is this still needed, or just for debugging?

Reply:
No  longer needed,  just for debugging.